_basics/install.md

```diff
@@ -38,5 +38,5 @@ To configure Marlin you can use the Arduino IDE or your [favorite text editor](h
 Follow one of these links to install Marlin:
 
 - [Installing Marlin with Arduino IDE](install_arduino.html) : To install Marlin 1.1.x or Marlin 2.0.x on AVR boards
-- [Installing Marlin with PlatformIO](install_platformio.html) : To install Marlin 2.0.x on any platform, including AVR
+- [Installing Marlin with PlatformIO](install_platformio.html) : Automated method to install Marlin 1.1.x or Marlin 2.0.x on any platform, including AVR
```

Comment:
The whole purpose of the auto_build script is to hide all the manual steps.

On the other hand, the mapping of the PlatformIO environment to the board is now in the PINS.H file so the hard part has been simplified.

The real question is: How do we organize it to be useful to our users?

Maybe
1. Change the text from "Automated method to install ..." to "Automated **and manual** methods to install ...
2. Add links to the current install_platformio.md pointing to a doc on the manual and a doc on running the auto_build script when not in Atom
3. Create the manual-method doc and the not-in-Atom doc

Reply:
I've done some rearrangement and cleanup on the install pages, to put the steps in chronological order, list the build/install steps just once, and to add links to Process Palette and the Atom documentation on installing packages. Also added links to the Re-Arm install page for manual installation and tried to match more of the contents between the two files.

PlatformIO has added a full interactive debugger, so I added a note about that.

For now the PlatformIO install page is limited to auto-install from Atom, and that's fine. The Re-Arm Install page can serve as the Manual Install page, using Re-Arm as the example. As we get more experience with builds we can add notes for any special cases we run into.

The next useful thing would be to add some images — not necessarily full screenshots, but the important visual details. A picture of the build results, etc.

I'd definitely like to add pages for VSCode and command-line PlatformIO too, and it would be great if we could implement auto-build for VSCode. We might also want to add instructions for Python and Gem installation for the needed dependencies. I still haven't installed TkInter.